packages/arb-token-bridge-ui/src/components/syncers/PendingTransactionsUpdater.tsx

```diff
@@ -39,27 +39,30 @@ export function PendingTransactionsUpdater(): JSX.Element {
     if (!arbTokenBridgeLoaded) return
     updateCctpTransactions()
     const pendingTransactions = actions.app.getPendingTransactions()
-    if (pendingTransactions.length) {
-      console.info(
-        `Checking and updating ${pendingTransactions.length} pending transactions' statuses`
-      )
 
-      // eslint-disable-next-line consistent-return
-      return Promise.all(
-        pendingTransactions.map((tx: Transaction) => getTransactionReceipt(tx))
-      ).then((txReceipts: (TransactionReceipt | null)[]) => {
-        txReceipts.forEach((txReceipt: TransactionReceipt | null, i) => {
-          if (!txReceipt) {
-            console.info(
-              'Transaction receipt not yet found:',
-              pendingTransactions[i]?.txID
-            )
-          } else {
-            arbTokenBridge?.transactions?.updateTransaction(txReceipt)
-          }
-        })
-      })
+    if (!pendingTransactions.length) {
```

Comment:
This change doesn't have to do anything with `walletAddress`, it's just a switch to an early return?